tests/unit/test_http_session.py

```diff
@@ -367,6 +367,26 @@ def test_proxy_ssl_context_uses_check_hostname(self):
             last_call = self.proxy_manager_fun.call_args[-1]
             self.assertIs(last_call['ssl_context'].check_hostname, True)
 
+    def test_proxy_ssl_context_does_not_use_check_hostname_if_ip_address(self):
+        cert = ('/some/cert', '/some/key')
+        proxies_config = {'proxy_client_cert': "path/to/cert"}
+        urls = ['https://1.2.3.4:5678',
+                'https://4.6.0.0',
+                'https://[FE80::8939:7684:D84b:a5A4%251]:1234',
+                'https://[FE80::8939:7684:D84b:a5A4%251]',
+                'https://[::1]:789']
+        for proxy_url in urls:
+            with mock.patch('botocore.httpsession.SSLContext'):
```

Comment:
It looks like there are some issues with your current Mock usage causing this test to fail. Can you take a look at that?

Reply:
I can see the tests are good now, until I change the import to `from botocore.utils import IPV6_ADDRZ_RE` instead of urllib3.
pytest is not able to import from utils somehow, whereas python can. I committed anyways as I wanted to see if this was something with my workspace.